documentation/cloud-manager-guide/admin/clusters/cluster-monitoring.md

```diff
@@ -0,0 +1,96 @@
+---
+keywords:
+title: Cluster Monitoring
+description: Cluster monitoring
+---
+# Cluster Monitoring
+
+The status of each cluster and all of its nodes can be monitored from the *Clusters* tab in the *Admin* section of Environment Operations Center. This guide outlines how to review the status and details of a cluster and how to review the status of all nodes within a cluster.
+
+## Getting started
+
+On the *Clusters* tab, each cluster is listed including its region, number of nodes, Kubernetes version, and status.
+
+Depending on the operational health of a cluster its status will be listed as:
+
+> **RL QUESTION: Requested definitions and confirmation for the following statuses**
+
+- Ready:
+- Running:
+- Warning:
+- Operational:
+
+![image description](images/cluster-overview.png)
+
+## View Cluster details
+
+To view further details about a cluster and its nodes, select the cluster name.
+
+![image description](images/select-name.png)
+
+The detailed view of a cluster provides the cluster general info and utilization status. A list of all nodes and their statuses are listed below the cluster details.
+
+![image description](images/cluster-details.png)
+
+### Cluster info
+
+General cluster details are listed under "Cluster Info" including:
+
+> **RL QUESTION: Requested definitions and confirmation for the following statuses**
+
+- Name:
+- Overall status: The current operational status of the cluster. This will either be "Ready", "Running", "Warning", or "Unavailable".
+- Region: The region where cluster the servers are located.
```

Comment:
I know this isn't final, but I think "the" and "cluster" might be reversed here?